tests/behat/behat_repository_ocis.php

```diff
@@ -95,38 +140,136 @@ private function get_password_for_user(string $username): string {
     }
 
     /**
-     * @Given :user has created a file :file in :space space
+     * @param string $user
+     * @param string $file
+     * @param string $body
+     *
+     * @return void
+     * @throws Exception
      */
-    public function user_has_created_a_file($user, $file, $space) {
-        if (strtolower($space) === "personal") {
-            $this->create_file_in_personal_space($user, $file);
+    private function create_file_in_personal_space(string $user, string $file, string $body): void {
+        $client = $this->get_client($user);
+        $response = $client->request('PUT', "/dav/files/$user/$file", $body);
+        if (!in_array($response['statusCode'], [201, 204])) {
+            throw new Exception("Error creating resource in ocis " . var_dump($response['statusCode']));
```

Comment:
```suggestion
            throw new Exception("Error creating resource in ocis personal space of user '$user', status was " . var_dump($response['statusCode']));
```

will it help to give more information in the exception message?